src/compas_fab/robots/robot.py

```diff
@@ -871,48 +1037,61 @@ def position_constraint_from_frame(self, frame_WCF, tolerance_position, group=No
         frame_WCF : :class:`compas.geometry.Frame`
             The frame from which we create position and orientation constraints.
         tolerance_position : float
-            The allowed tolerance to the frame's position. (Defined in the
-            robot's units)
-        group: str
+            The allowed tolerance to the frame's position (defined in the
+            robot's units).
+        group: str, optional
             The planning group for which we specify the constraint. Defaults to
             the robot's main planning group.
 
+        Returns
+        -------
+        :class:`PositionConstraint`
+
+        See Also
+        --------
+        * :meth:`PositionConstraint.from_box`
+        * :meth:`PositionConstraint.from_mesh`
+        * :meth:`PositionConstraint.from_sphere`
+
         Examples
         --------
         >>> frame = Frame([0.4, 0.3, 0.4], [0, 1, 0], [0, 0, 1])
         >>> tolerance_position = 0.001
         >>> robot.position_constraint_from_frame(frame, tolerance_position)
         PositionConstraint('ee_link', BoundingVolume(2, Sphere(Point(0.400, 0.300, 0.400), 0.001)), 1.0)
-
-        Notes
-        -----
-        There are many other possibilities of how to create a position and
-        orientation constraints. Checkout :class:`compas_fab.robots.PositionConstraint`
-        and :class:`compas_fab.robots.OrientationConstraint`.
-
         """
-
         ee_link = self.get_end_effector_link_name(group)
         sphere = Sphere(frame_WCF.point, tolerance_position)
         return PositionConstraint.from_sphere(ee_link, sphere)
 
     def constraints_from_frame(self, frame_WCF, tolerance_position, tolerances_axes, group=None):
-        """Returns a position and orientation constraint on the group's end-effector link.
+        """Returns a position and an orientation constraint on the group's end-effector link.
 
         Parameters
         ----------
         frame_WCF: :class:`compas.geometry.Frame`
             The frame from which we create position and orientation constraints.
         tolerance_position: float
-            The allowed tolerance to the frame's position. (Defined in the
-            robot's units)
+            The allowed tolerance to the frame's position (defined in the
+            robot's units).
         tolerances_axes: list of float
-            Error tolerances ti for each of the frame's axes in radians. If only
+            Error tolerances *ti* for each of the frame's axes in radians. If only
             one value is passed it will be uses for all 3 axes.
-        group: str
+        group: str, optional
             The planning group for which we specify the constraint. Defaults to
             the robot's main planning group.
 
+        Returns
+        -------
+        List of a :class:`PositionConstraint` and an :class:`OrientationConstraint`.
```

Comment:
Why not use the base class, and say 
```
list of :class:`Constraint`
```
?

Reply:
Note to self: Fixed on local head.